Sources/LiveViewNative/Property Wrappers/Attribute.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  Attribute.swift
+//  
+//
+//  Created by Carson Katri on 2/28/23.
+//
+
+import SwiftUI
+import LiveViewNativeCore
+
+@propertyWrapper
+public struct Attribute<T>: DynamicProperty where T: AttributeDecodable {
+    @ObservedElement private var element
+    private let name: AttributeName
+    private let defaultValue: T?
+
+    public init(wrappedValue: T? = nil, _ name: AttributeName) {
+        self.name = name
+        self.defaultValue = wrappedValue
+    }
+    
+    public init(wrappedValue: T? = nil, _ name: String) {
+        self.name = AttributeName(name: name)
+        self.defaultValue = wrappedValue
+    }
```

Comment:
I don't think this initializer is necessary, since AttributeName conforms to ExpressibleByStringLiteral